js/views/modals/listingDetail/Listing.js

```diff
@@ -528,6 +529,16 @@ export default class extends BaseModal {
       this.$reviews = this.$('.js-reviews');
       this.$reviews.append(this.reviews.render().$el);
 
+      if (!this.model.isOwnListing) {
+        if (this.socialBtns) this.socialBtns.remove();
+        this.socialBtns = this.createChild(SocialBtns, {
+          targetID: this.vendor.peerID,
+          stripClasses: 'btnStrip clrSh3',
```

Comment:
It may not be a bad idea to pass in stripClasses and btnClasses in as state, rather than options. The benefit of that is you could dynamically change them then.